### PR TITLE
Update library

### PR DESCRIPTION
**Pull request by addyi**
- Disable jetifier
- Update dependencies
- Update library version to 1.2.0

### CODE REVIEW

File: library/build.gradle [hunk 1]
```diff
@@ -3,35 +3,31 @@ apply plugin: 'kotlin-android'
...
-  defaultConfig {
-    minSdkVersion 15
-    targetSdkVersion 28
-    versionCode 1
-    versionName "1.1.4"
+    defaultConfig {
+        minSdkVersion 15
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.2.0"
```
Comment (androideveloper) [inline library/build.gradle:11]: You probably also want to update readme file with 
`implementation 'com.github.teresaholfeld:Stories:1.2.0'`

File: app/build.gradle [hunk 2]
```diff
@@ -2,34 +2,29 @@ apply plugin: 'com.android.application'
...
-    versionCode 1
-    versionName "1.0"
-    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-  }
-  buildTypes {
-    release {
-      minifyEnabled false
-      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+    compileSdkVersion 30
+    buildToolsVersion '28.0.3'
```
Comment (androideveloper) [inline app/build.gradle]: Please remove this line

> The specified Android SDK Build Tools version (28.0.3) is ignored, as it is below the minimum supported version (29.0.2) for Android Gradle Plugin 4.1.3.

File: library/build.gradle [hunk 3]
```diff
@@ -3,35 +3,31 @@ apply plugin: 'kotlin-android'
 apply plugin: 'com.github.dcendents.android-maven'
 
 android {
-  compileSdkVersion 28
-  buildToolsVersion '28.0.3'
-  group = "com.teresaholfeld.stories"
+    compileSdkVersion 30
+    buildToolsVersion '28.0.3'
```
Comment (androideveloper) [inline library/build.gradle]: Please remove this line

> The specified Android SDK Build Tools version (28.0.3) is ignored, as it is below the minimum supported version (29.0.2) for Android Gradle Plugin 4.1.3.

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
-#Mon Sep 10 10:57:05 WEST 2018
+#Wed Mar 31 12:35:12 CEST 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
```
Comment (androideveloper) [inline gradle/wrapper/gradle-wrapper.properties]: bin -> all

Diff https://stackoverflow.com/a/25456329/1635488

**Review by androideveloper — CHANGES_REQUESTED** [review]
Done! Added a couple of comments below

File: library/build.gradle (hunk 1 above)
Comment (addyi) [reply library/build.gradle:11]: done with c3bd9fc

File: app/build.gradle (hunk 2 above)
Comment (addyi) [reply app/build.gradle]: done with 4f848a5

File: library/build.gradle (hunk 3 above)
Comment (addyi) [reply library/build.gradle]: done with 4f848a5

File: gradle/wrapper/gradle-wrapper.properties (hunk 4 above)
Comment (addyi) [reply gradle/wrapper/gradle-wrapper.properties]: done with 73662f1

File: build.gradle [hunk 5]
```diff
@@ -1,14 +1,14 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-  ext.kotlin_version = '1.3.21'
+  ext.kotlin_version = '1.4.32'
   repositories {
     google()
     jcenter()
```
Comment (omatuzenko-stocard) [inline build.gradle]: Is it possible to remove jcenter here?

**Review by omatuzenko-stocard — APPROVED** [review]
Just one remark about jcenter and travis build check fail if there is something we need to check

File: build.gradle (hunk 5 above)
Comment (androideveloper) [reply build.gradle]: There is one more below, would be great to also remove it
I checked, seems that `com.github.dcendents:android-maven-gradle-plugin` is available on mavencentral. But it's deprecated project anyway, so maybe we can remove it?

**Review by androideveloper — APPROVED** [review]
Looks good! Let's see if they accept the changes ^^

**Pull request merged**